openvisualizer/main.py

```diff
@@ -43,6 +43,27 @@
 coloredlogs.install(level='WARNING', logger=log, fmt='%(asctime)s [%(name)s:%(levelname)s] %(message)s',
                     datefmt='%H:%M:%S')
 
+# ============================ functions =======================================
+
```

Comment:
this function is not necessary.
see https://stackoverflow.com/questions/15753701/how-can-i-pass-a-list-as-a-command-line-argument-with-argparse